CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.54.2 / 2024-09-02
+
+* [BUGFIX] Scraping: Fixed significant performance regression in OM Text parsing related to Created Timestamp parsing; that logic has been moved behind an experimental `created-timestamp-zero-ingestion` flag as it should be. #14809
```

Comment:
I'm not sure if this is a correct wording. We didn't fix regression. It is known naive implementation we suppose to do behind a feature flag and fix == moving behind a feature flag.

Feedback welcome.

Reply:
ack, we're more so isolating the regression behind the flag. I'll still have to work on an actual fix